src/CMakeLists.txt

```diff
@@ -34,6 +34,34 @@ peloton_default_properties(peloton-proto)
 
 add_subdirectory(${PROJECT_SOURCE_DIR}/third_party/libpg_query/ libpg_query.a)
 
+##################################################################################
+
+# TODO: Guess this should be moved into a module?
+# --[ tensorflow
+# Add library to be linked
+find_library(TFlowC
+        NAMES tensorflow
+        PATHS "/usr/local/lib")
+set(Peloton_LINKER_LIBS "${TFlowC}" ${Peloton_LINKER_LIBS}) # note, crucial to prepend!
```

Comment:
Yes it's better to create a separate module file for this. But even though you don't want to do that because it's short, at least you should move this to cmake/Dependencies.cmake instead of here.